docs/guides/routing.mdx

```diff
@@ -0,0 +1,45 @@
+---
+description: Learn how Clerk handles routing in your application.
+---
+
+# Routing in Clerk
+
+Clerk uses [React Router](https://reactrouter.com/en/main) behind the scenes in order to handle routing in Clerk's [Account Portal](/docs/account-portal/overview) and [components](/docs/components/overview).
```

Comment:
We don't actually use React Router. The implementation is similar to React Router, but our routers are custom-built at the moment. Not sure where this was communicated, sorry!

Reply:
from roy in the thread that we were all talking in, and no one corrected him 

https://clerkinc.slack.com/archives/C046VRJPCNS/p1712858410829219?thread_ts=1712857764.014139&cid=C046VRJPCNS